buildSrc/src/main/java/com/uber/okbuck/core/dependency/ExternalDependency.java

```diff
@@ -107,56 +237,36 @@ private ExternalDependency(
       @Nullable String version,
       @Nullable String classifier,
       File depFile,
-      boolean isLocal) {
-    this.group = group;
+      boolean isLocal,
+      ExternalExtension externalExtension) {
+
+    // TODO find a better solution when group ends with 'buck'
+    // which can cause it to clash with BUCK files generated at that path
+    if (group.contains(".buck")) {
```

Comment:
is this just for the prebuilt buck binaries we use?

Reply:
yes, currently. since we will have generated buck files named `BUCK` at this path, group ending with `buck` would be problematic.

Reply:
this will go away. working on moving buck download as a special tool task